### PR TITLE
GH-95 logging to file & standard out.

### PR DESCRIPTION
**Pull request by dsclassen**
Works for log messages originating in `loopDHS`. Need to add to other models/files

### CODE REVIEW

File: bin/loopDHS [hunk 1]
```diff
@@ -25,9 +27,16 @@ from pydhsfw.automl import AutoMLPredictRequest, AutoMLPredictResponse
 from pydhsfw.jpeg_receiver import JpegReceiverImagePostRequestMessage
 from pydhsfw.axis import AxisImageRequestMessage, AxisImageResponseMessage
 
-#_logger = logging.getLogger(__name__)
+# logging setup
```
Comment (gcmullen) [inline bin/loopDHS:30]: Better to remove commented out lines of code when checking in. You can always easily get them back using history if you need them.

Comment (dsclassen) [reply bin/loopDHS:30]: I added a small code comment just for the purposes of delineating the lines of code where logging is being configured.

Comment (gcmullen) [reply bin/loopDHS:30]: Riiiiight.... I missed that. Oops. Thumbs up!

**Review by gcmullen — APPROVED** [review]
Hooray!

**Pull request merged**